Jenkinsfile

```diff
@@ -275,7 +275,8 @@ def define_browserstack_tests() {
       withEnv([
         "BROWSER=${it.browser}",
         "BROWSERSTACK_CONFIGURATION_OPTIONS=${it.config}",
-        'BROWSERSTACK=true'
+        'BROWSERSTACK=true',
+        "BROWSERSTACK_BUILD_NAME=Design system (Jenkins) ${env.BRANCH_NAME}_${getSha()}"
```

Comment:
As `DOCKER_TAG=${env.BRANCH_NAME}_${getSha()}` you could arguably do ` "BROWSERSTACK_BUILD_NAME=Design system (Jenkins) ${env.DOCKER_TAG}"`. But this is a pretty short-lived fix so 🤷‍♂️

Reply:
forcing a re-run here: https://github.com/citizensadvice/design-system/pull/1239
looks like a goer: https://jenkins.devops.citizensadvice.org.uk/blue/organizations/jenkins/design-system/detail/PR-1239/1/pipeline

Reply:
it failed but only because no components could  load on iPhone IIX: The Last iPhone.